.changelog/20251219073121_ck_19483.md

```diff
@@ -0,0 +1,10 @@
+---
+type: Fix
+scope:
+  - ckeditor5-engine
+  - ckeditor5-undo
+closes:
+  - https://github.com/ckeditor/ckeditor5/issues/19483
+---
+
+Fixed an issue where `RootAttributeOperation` was incorrectly transformed to `NoOperation` when the new value was the same as in the operation it was transformed by. This fixes a bug where undoing root attribute changes would sometimes fail.
```

Comment:
This should be user-facing without technical details. What bug will the user see fixed?

Reply:
I simplified it a little bit.